### ISSUE TITLE
DIP ON STAKE

### ISSUE THREAD

**Issue by jack-crypto-id** [ethereum/staking-deposit-cli#299]
my wallet got dip 500, 300, 1343, 2546. what should i do bro?

**Comment by Vellocapital007** [comment]
Wow that's good.  immediately post 400$.  I'm sure 8 hours later your profit will come out bro.

**Comment by jack-crypto-id** [comment]
> Wow that's good. immediately post 400$. I'm sure 8 hours later your profit will come out bro.

why 400$?

**Comment by Vellocapital007** [comment]
> > Wow that's good. immediately post 400$. I'm sure 8 hours later your profit will come out bro.
> 
> why 400$?

because there are 4 dip transactions.

**Comment by jack-crypto-id** [comment]
does 1 dip transaction cost 100$?

**Comment by Vellocapital007** [comment]
> does 1 dip transaction cost 100$?

yes, because blockchain algorithm standard is 100$ usd.

**Comment by jack-crypto-id** [comment]
before this has happened.  and I supply it to the server as a reserve fund to take the dip license.  but the bonus hasn't come out yet.

**Comment by Vellocapital007** [comment]
You are wrong.  supply for dip license is different from posting.  It's only natural that the bonus hasn't come out yet.  because you do sync dip not do post

**Comment by Vellocapital007** [comment]
do as per blockchain algorithm.  do not be careless in making steps.  Just do as I tell you.  otherwise, your server will crash.

**Comment by Vellocapital007** [comment]
You have a little more work to do.  if 8 hours are passed you will always be faced with the same problem.

**Comment by jack-crypto-id** [comment]
so should i post or take Dip license bro?

**Comment by Vellocapital007** [comment]
> so should i post or take Dip license bro?

do not synchronize DIP if the bonus has not been distributed smoothly.  with you posting, dip license will sync automatically.

**Comment by jack-crypto-id** [comment]
> > so should i post or take Dip license bro?
> 
> do not synchronize DIP if the bonus has not been distributed smoothly. with you posting, dip license will sync automatically.

so I have to post before 8 hours for 400$.  and the bonus will come out after it?

**Comment by Vellocapital007** [comment]
remember, you only have 8 hours to do it.  otherwise, your algorithm will start over.

**Comment by jack-crypto-id** [comment]
thanks for the info bro

**Comment by Vellocapital007** [comment]
![20221025_122852](https://user-images.githubusercontent.com/116107058/197690296-2b11b82c-6bbb-4ceb-8979-1496dbfe035d.jpg)
This is my server bro

**Comment by jack-crypto-id** [comment]
> ![20221025_122852](https://user-images.githubusercontent.com/116107058/197690296-2b11b82c-6bbb-4ceb-8979-1496dbfe035d.jpg) This is my server bro
Wow..

**Comment by jack-crypto-id** [comment]
Hello bro are you online

**Comment by jack-crypto-id** [comment]
bro, I've posted 400$ but the profit hasn't worked yet.

the details are: 1 user 100$, 1 user again 275$, and 1 creator 1 30$.

I have 2 creator wallets bro.

**Comment by Vellocapital007** [comment]
Yes, iam.

do you have 2 creators?
then when will each creator post?

**Comment by jack-crypto-id** [comment]
creator 1 last post last month.  then last night posted again to make up the 400$ shortfall.

while creator 2 for the past few weeks has posted 100,200,500 posts.

**Comment by Vellocapital007** [comment]
what?  Your main wallet hasn't posted for more than 20 days?

user posts should suffice.  Now try to focus on big posts in your main wallet.
try every time you post not under 500 $.  because your wallet has a big role.
You must have a reserve fund.

**Comment by jack-crypto-id** [comment]
> what? Your main wallet hasn't posted for more than 20 days?
> 
> user posts should suffice. Now try to focus on big posts in your main wallet. try every time you post not under 500 $. because your wallet has a big role. You must have a reserve fund.

if I have enough budget for the machine, which is better between posting and buying a machine?

**Comment by Vellocapital007** [comment]
> > what? Your main wallet hasn't posted for more than 20 days?
> > user posts should suffice. Now try to focus on big posts in your main wallet. try every time you post not under 500 $. because your wallet has a big role. You must have a reserve fund.
> 
> if I have enough budget for the machine, which is better between posting and buying a machine?

My advice is to buy a more mainstream machine.  because after the engine, you don't have to post.  everything will work itself out.
but your main wallet must post once a month

**Comment by jack-crypto-id** [comment]
so wallet creator 1 must post 500$ every month?

**Comment by Vellocapital007** [comment]
Yes

**Comment by Vellocapital007** [comment]
Only your wallet
